docs/initial-sync.md

```diff
@@ -0,0 +1,58 @@
+# File copy based initial sync
+
+!!! admonition "Version added: [7.0.21-12](release_notes/7.0.21-12.md)"
+
+When a new member joins the replica set, it receives the data from the existing replica set node via the initial sync. 
+
+In [Percona Server for MongoDB Pro](psmdb-pro.md), you can choose a file copy-based initial sync for a new node. You must have WiredTiger defined as the storage.
+
+File copy-based initial sync method is a physical copying of the data files from the source to the target. It is much faster than the default [logical initial sync :octicons-link-external-16:](https://www.mongodb.com/docs/manual/core/replica-set-sync/#logical-initial-sync-process), which is especially beneficial in heavy write environments. Using this sync method speeds up cluster scaling and increases restore performance. 
+
+File copy-based initial sync implementation is compatible with that of MongoDB Advanced and has the same [configuration parameters](#configuration-parameters).
+
+To select the initial sync method, specify the following configuration in the configuration file for the target node:
+
+```yaml
+setParameter:
+  initialSyncMethod: fileCopyBased
+```
+
+You can only set this configuration at startup.
```

Comment:
consider this:
> This server parameter can only be set at startup

Reply:
I'd leave the active voice. Replaced configuration with server parameter.